src/current/v23.2/known-limitations.md

```diff
@@ -165,7 +171,7 @@ In cases where the partition definition includes a comparison with `NULL` and a
 
 {% include {{page.version.version}}/known-limitations/sql-cursors.md %}
 
-### `SELECT FOR UPDATE` locks are dropped on lease transfers  and range splits/merges
+### `SELECT ... FOR UPDATE` and `SELECT ... FOR SHARE` locks are dropped on lease transfers and range splits/merges
```

Comment:
Should we clarify that this is only the case for serializable transactions and that it does not apply to read committed transactions?

Reply:
I've made this clearer. Also realized that this heading duplicates another heading on the KLs doc, so I did some reorganizing.